cannot-connect-to-database-server-from-a-data-link-server.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Cannot Connect to the Database Server from the {{site.dls}}
+description: Check your connection string if you receive the &quot;Cannot connect to database&quot; error.
+type: troubleshooting
+page_title:  Unsuccessful Connection Between {{site.dls}} and Database
+slug: cannot-connect-to-database-server-from-a-data-link-server
+position:
+tags: data connector, data link server, data, sql, everlive, backend, connection
+teampulseid: 
+ticketid: 1005975, 950185
+pitsid:
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Data</td>	
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Data Connectors</td>	
+  </tr>
+</table>
+
+## Description
+When configuring a {{site.dc}} to a database that requires the {{site.dls}}, an error occurs after clicking the **Test** button.
+
+At the same time, opening the {{site.dls}} URL in a web browser returns *Result: OK*.
+
+## Error Message
+`Data Link Server Error: Cannot connect to database. [Telerik Platform] The connection to the host [hostname], named instance [instance name] failed. Error: [message]. Verify the server and instance names and check that no firewall is blocking UDP traffic to port 1434.  For SQL Server 2005 or later, verify that the SQL Server Browser Service is running on the host.`
+
+## Cause
+The error signifies that the {{site.dls}} cannot connect to the database even though the connection between {{site.tp}} and the {{site.dls}} is successful.
+
+## Solution
+Ensure that the supplied connection string the correct format for the database:
```

Comment:
connection string uses the correct ...
or
connection string is in the correct ...